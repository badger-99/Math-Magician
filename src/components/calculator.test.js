import { render } from '@testing-library/react';
import Calculator from './calculator';

test('App should match the snapshot', () => {
  const component = render(<Calculator />);
  expect(component).toMatchSnapshot();
});
