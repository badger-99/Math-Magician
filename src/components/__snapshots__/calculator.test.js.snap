// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content calculator"
      >
        <h3>
          Let's do some math!
        </h3>
        <div
          class="border"
          id="grid"
        >
          <div
            class="border"
            id="display"
          >
            <h2>
              0
            </h2>
          </div>
          <button
            id="id0"
            name="AC"
            type="button"
          >
            AC
          </button>
          <button
            id="id1"
            name="+/-"
            type="button"
          >
            +/-
          </button>
          <button
            id="id2"
            name="%"
            type="button"
          >
            %
          </button>
          <button
            id="id3"
            name="รท"
            type="button"
          >
            รท
          </button>
          <button
            id="id4"
            name="7"
            type="button"
          >
            7
          </button>
          <button
            id="id5"
            name="8"
            type="button"
          >
            8
          </button>
          <button
            id="id6"
            name="9"
            type="button"
          >
            9
          </button>
          <button
            id="id7"
            name="x"
            type="button"
          >
            x
          </button>
          <button
            id="id8"
            name="4"
            type="button"
          >
            4
          </button>
          <button
            id="id9"
            name="5"
            type="button"
          >
            5
          </button>
          <button
            id="id10"
            name="6"
            type="button"
          >
            6
          </button>
          <button
            id="id11"
            name="-"
            type="button"
          >
            -
          </button>
          <button
            id="id12"
            name="1"
            type="button"
          >
            1
          </button>
          <button
            id="id13"
            name="2"
            type="button"
          >
            2
          </button>
          <button
            id="id14"
            name="3"
            type="button"
          >
            3
          </button>
          <button
            id="id15"
            name="+"
            type="button"
          >
            +
          </button>
          <button
            id="id16"
            name="0"
            type="button"
          >
            0
          </button>
          <button
            id="id17"
            name="."
            type="button"
          >
            .
          </button>
          <button
            id="id18"
            name="="
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content calculator"
    >
      <h3>
        Let's do some math!
      </h3>
      <div
        class="border"
        id="grid"
      >
        <div
          class="border"
          id="display"
        >
          <h2>
            0
          </h2>
        </div>
        <button
          id="id0"
          name="AC"
          type="button"
        >
          AC
        </button>
        <button
          id="id1"
          name="+/-"
          type="button"
        >
          +/-
        </button>
        <button
          id="id2"
          name="%"
          type="button"
        >
          %
        </button>
        <button
          id="id3"
          name="รท"
          type="button"
        >
          รท
        </button>
        <button
          id="id4"
          name="7"
          type="button"
        >
          7
        </button>
        <button
          id="id5"
          name="8"
          type="button"
        >
          8
        </button>
        <button
          id="id6"
          name="9"
          type="button"
        >
          9
        </button>
        <button
          id="id7"
          name="x"
          type="button"
        >
          x
        </button>
        <button
          id="id8"
          name="4"
          type="button"
        >
          4
        </button>
        <button
          id="id9"
          name="5"
          type="button"
        >
          5
        </button>
        <button
          id="id10"
          name="6"
          type="button"
        >
          6
        </button>
        <button
          id="id11"
          name="-"
          type="button"
        >
          -
        </button>
        <button
          id="id12"
          name="1"
          type="button"
        >
          1
        </button>
        <button
          id="id13"
          name="2"
          type="button"
        >
          2
        </button>
        <button
          id="id14"
          name="3"
          type="button"
        >
          3
        </button>
        <button
          id="id15"
          name="+"
          type="button"
        >
          +
        </button>
        <button
          id="id16"
          name="0"
          type="button"
        >
          0
        </button>
        <button
          id="id17"
          name="."
          type="button"
        >
          .
        </button>
        <button
          id="id18"
          name="="
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
